problemtools/languages.yaml

```diff
@@ -141,6 +141,13 @@ javascript:
     compile: '/usr/bin/js24 -c {files}'
     run: '/usr/bin/js24 "{mainfile}"'
 
+kotlin:
+    name: 'Kotlin'
+    priority: 250
+    files: '*.kt *.ktm *.kts'
+    compile: '/usr/bin/kotlinc -jvm-target 1.8 {files} -include-runtime -d {binary}.jar'
+    run: '/usr/bin/kotlin -J-XX:+UseSerialGC -J-XX:ReservedCodeCacheSize=4m -J-XX:MaxMetaspaceSize=8m -J-XX:CompressedClassSpaceSize=8m -J-Xss5m -J-Xms5m -J-Xmx256m {binary}.jar'
```

Comment:
Can you explain what all the -J options that are not specified for java do (and why they're are relevant to Kotlin)?

Why do you include the full runtime if you do run kotlin later? In DOMjudge (https://github.com/DOMjudge/domjudge/blob/master/sql/files/defaultdata/kt/run) we just run kotlinc over all the sources and then run kotlinc with the same options as for Java (prefixed with -J of course).